### DISCUSSION TITLE
How to get messages in realtime

### DISCUSSION THREAD

**Discussion by sarada-syon** [xmppjs/xmpp.js#894] [Q&A] [answered]
By using @xmpp/client i am able to send messages. But not getting how to receive messages. Can anyone please suggest me any event. Thanks

**Comment by sonnyp** [comment, marked as answer]
See https://github.com/xmppjs/xmpp.js/blob/master/packages/client/example.js

If you're not getting any message - that's probably because you're not sending a presence.
